demo/common.html

```diff
@@ -21,4 +22,6 @@
       button.addEventListener("click", fnc);
       return button;
     }
+
+    var randomUserUrl = document.location.protocol === 'https:' ? 'https://randomuser.me/api/' : 'http://api.randomuser.me/';
```

Comment:
Could’ve used `'//randomuser.me/api/' to avoid this variable, but doesn’t really matter.